e2e/scenario_test.go

```diff
@@ -42,15 +46,19 @@ func TestAll(t *testing.T) {
 	t.Run("ubuntu1804", Scenario_ubuntu1804)
 	t.Run("ubuntu1804-azurecni", Scenario_ubuntu1804_azurecni)
 	t.Run("ubuntu1804-chrony-restarts", Scenario_ubuntu1804ChronyRestarts)
+	t.Run("ubuntu1804-containerd-version", Scenario_ubuntu1804HasRightContainerdVersion)
+	t.Run("ubuntu1804-runc-version", Scenario_ubuntu1804HasRightRuncVersion)
 	t.Run("ubuntu1804-gpu", Scenario_ubuntu1804gpu)
 	t.Run("ubuntu1804-gpu-azurecni", Scenario_ubuntu1804gpu_azurecni)
 	t.Run("ubuntu2204", Scenario_ubuntu2204)
 	t.Run("ubuntu2204-airgap", Scenario_ubuntu2204AirGap)
 	t.Run("ubuntu2204-arm64", Scenario_ubuntu2204ARM64)
 	t.Run("ubuntu2204-artifact-streaming", Scenario_ubuntu2204ArtifactStreaming)
 	t.Run("ubuntu2204-chrony-restarts", Scenario_ubuntu2204ChronyRestarts)
-	t.Run("ubuntu2204-containerd-url", Scenario_ubuntu2204ContainerdURL)
-	t.Run("ubuntu2204-containerd-version", Scenario_ubuntu2204ContainerdVersion)
+	t.Run("ubuntu2204-containerd-version", Scenario_ubuntu2204HasRightContainerdVersion)
+	t.Run("ubuntu2204-runc-version", Scenario_ubuntu2204HasRightRuncVersion)
+	t.Run("ubuntu2204-containerd-override", Scenario_ubuntu2204ContainerdURL)
+	t.Run("ubuntu2204-containerd-cur-ver", Scenario_ubuntu2204ContainerDHasCurrentVersion)
```

Comment:
dome test name are suing `D` some `d` for containerd.

Reply:
Will fix.